src/codegen.cpp

```diff
@@ -5964,7 +5961,7 @@ static void init_julia_llvm_env(Module *m)
         &pow,
```

Comment:
I don't think that is an issue.

1. These should be inline functions that calls the C `::pow`, `::powf`, `::powl`.
2. For C++ implementations that provide these it shouldn't be allowed to overwrite the existing `::pow(double, double)` symbol so the it should always be the C one.
3. At least in the generic linux binary `cglobal(:pow)` and `ccall(:pow, ..)` points to the `libm` version.